Makefile

```diff
@@ -12,6 +12,7 @@ fast-cover:
 	go test -cover -tags="integration" $(TESTFLAGS)
 
 generate:
+	sed -i -e 's/package internal;/package gopilosa_pbuf;/g' gopilosa_pbuf/public.proto
```

Comment:
I don't understand why this needs to be in the Makefile. Isn't it sufficient to make the change once to the .proto file and then run `make generate`?

Reply:
Yeah, but we copy the proto file from the Pilosa repo, and we may need to update it manually. We may forget to update the proto file too.

Reply:
oh, I see - good idea. Can you add a comment to make that clear?

Reply:
Sure, updated.